.github/workflows/docker-build.yml

```diff
@@ -191,6 +191,10 @@ jobs:
         uses: helm/kind-action@v1
         with:
           install_only: true
+      - name: Install astria cli (rust)
+        run: just install-cli
+      - name: Fetch and install celestia-appd
+        run: just get-celestia-appd v1.9.0 Linux x86_64
```

Comment:
This could be avoided if we make celestia-local into a stateful set. We could then `kubectl execute` from inside the celestia-local container, making this step unnecessary.

Reply:
Yes, this indeed has that issue, which running directly inside a stateful set wouldn't have. Followup sounds good. I will create an issue.